concourse/tasks/validate-image-e2etest-result.yaml

```diff
@@ -0,0 +1,20 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/gcp-guest/validate-e2etest
```

Comment:
what's special about this image? if it's just running a shell script, would some common public image work? where is the image definition?

Reply:
This should be similar to validate integration test. But I am not sure if we could reuse the validate-integtest image to make it work for e2e test. Consider we both use junit as report, probably, we could adapt validate-integtest image to use for e2e test